modular_skyrat/modules/customization/datums/dna.dm

```diff
@@ -14,6 +16,16 @@
 	features = species.get_random_features()
 	mutant_bodyparts = species.get_random_mutant_bodyparts(features)
 
+/datum/dna/proc/update_body_size()
+	if(!holder || current_body_size == features["body_size"])
+		return
+	var/change_multiplier = features["body_size"] / current_body_size
+	//We update the translation to make sure our character doesn't go out of the southern bounds of the tile
+	var/translate = ((change_multiplier-1) * 32)/2
+	holder.transform = holder.transform.Scale(change_multiplier, change_multiplier)
+	holder.transform = holder.transform.Translate(0, translate)
+	current_body_size = features["body_size"]
```

Comment:
Perhaps we could shift their pixel_y up? Based on size?

Reply:
That's what is happening